src/AppBundle/Form/PostType.php

```diff
@@ -39,7 +39,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         //     $builder->add('title', null, array('required' => false, ...));
 
         $builder
-            ->add('title', null, array('label' => 'label.title'))
+            ->add('title', null, array(
+                'attr' => array('autofocus' => true),
```

Comment:
@rasanga, I agree too, but as @voronkovich says, just follows the existing code style. https://github.com/symfony/symfony-demo/blob/master/src/AppBundle/Form/PostType.php#L45

Another way is using javascripts `$('#app_post_title').focus();` within edit and new templates, but I still prefer this PR. Is simple and without dependencies.